src/Resource.php

```diff
@@ -205,11 +205,11 @@ protected function mergeAttributes(array $attributes)
      *
      * @return Relationship[]
      */
-    public function getRelationships()
+    public function getRelationships($filter = true)
```

Comment:
Can you add a separate method here? Maybe `getFilteredRelationships()`? That's much nicer than having to call it with a boolean parameter that doesn't imply what it's about...